hw/ip/keymgr/doc/_index.md

```diff
@@ -47,7 +47,9 @@ They differ only in how software chooses to deploy the outputs.
 ## Key Manager State
 
 The key manager working state (red boxes in the functional model) represents both the current state of the key manager as well as its related secret material.
-Each state, when valid, supplies its secret material as the "key" input to a KMAC operation.
+Each valid state (`Initialized` / `CreatorRootKey` / `OwnerIntermediateKey` / `OwnerRootKey`), supplies its secret material as the "key" input to a KMAC operation.
+Invalid states, such as `Reset / Disabled` on the other hand, either do not honor operation requests, or supplies random data when invoked.
```

Comment:
for reset, I think we just update status to fail without sending data to kmac, right?
The spec mentions "Until the initialize command is invoked, the key manager rejects all other software commands" for reset

Reply:
yes that's what i meant by "do not honor operation requests".

Reply:
you're right, I missed that one.

Reply:
i'm going to try and clean this up some more.  Give me a bit.

Reply:
@weicaiyang i'm probably going to revamp how some of this error stuff is handled. It's a bit too messy right now.

Reply:
sounds good, thanks for the heads-up.